net-dns/coredns/coredns-1.11.2.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fcaps go-module multiprocessing systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="CoreDNS is a DNS server that chains plugins"
+HOMEPAGE="https://github.com/coredns/coredns"
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/coredns/coredns.git"
+else
+	SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
@rahilarious Somehow we used an unofficial tag of coredns since https://github.com/coredns/coredns/releases/tag/v1.11.2 currently doesn't exist and when [I asked upstream](https://github.com/coredns/coredns/pull/6476#issuecomment-2017144142) they said it's [not released yet](https://github.com/coredns/coredns/pull/6476#issuecomment-2017782380). For the time being I've updated SRC_URI in ee1b98b319fe39920e33b11dd291261118176305.

Reply:
what the F!! They must've pulled it out. I can assure it was released even checkout repology https://repology.org/project/coredns/versions . I even have the email from github that I get on every release (watch). Forwarding you

Reply:
Thanks, yeah I received the release email too.

Reply:
When I first noticed the issue it seemed that coredns issues were not available on github, but anyway yeah they deleted the tag intentionally:

https://github.com/coredns/coredns/issues/6454#issuecomment-1996599800